addons/l10n_mx/migrations/2.1/post-migrate.py

```diff
@@ -0,0 +1,11 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, Command, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    debit_tag = env.ref("l10n_mx.tag_debit_balance_account")
+    credit_tag = env.ref("l10n_mx.tag_credit_balance_account")
+    mx_companies = env['res.company'].search([('account_fiscal_country_id', '=', env.ref('base.mx').id)])
+    accounts_102 = env['account.account'].search([('code', '=like', '102%'), ('company_id', 'in', mx_companies.ids)])
+    accounts_102.tag_ids = [Command.unlink(credit_tag.id), Command.link(debit_tag.id)]
```

Comment:
I double-checked with Laurent, this is too risky to do: we could end up changing wrong 102 accounts. The users could have changed accounts.
The only safe way is to *only* migrate our data `cuenta102_02` and let the user handle his other 102 accounts by hand.
Generally speaking searching company by fiscal country is not safe either, better use the chart_template, but in this case I think you can just re-use the snippet I suggested last review ? :thinking: